proteuslib/unit_models/reverse_osmosis_0D.py

```diff
@@ -823,29 +875,29 @@ def eq_equal_flow_vol_interface_io(b, t, x):
         @self.Constraint(self.flowsheet().config.time)
         def eq_recovery_vol_phase(b, t):
             return (b.recovery_vol_phase[t, 'Liq'] ==
-                    b.properties_permeate[t].flow_vol_phase['Liq'] /
+                    b.permeate_side.properties_permeate[t].flow_vol_phase['Liq'] /
                     b.feed_side.properties_in[t].flow_vol_phase['Liq'])
 
         @self.Constraint(self.flowsheet().config.time,
                          self.config.property_package.component_list)
         def eq_recovery_mass_phase_comp(b, t, j):
             return (b.recovery_mass_phase_comp[t, 'Liq', j] ==
-                    b.properties_permeate[t].flow_mass_phase_comp['Liq', j] /
+                    b.permeate_side.properties_permeate[t].flow_mass_phase_comp['Liq', j] /
                     b.feed_side.properties_in[t].flow_mass_phase_comp['Liq', j])
 
         @self.Constraint(self.flowsheet().config.time,
                          self.solute_list)
         def eq_rejection_phase_comp(b, t, j):
             return (b.rejection_phase_comp[t, 'Liq', j] ==
-                    1 - (b.properties_permeate[t].conc_mass_phase_comp['Liq', j] /
+                    1 - (b.permeate_side.properties_permeate[t].conc_mass_phase_comp['Liq', j] /
                          b.feed_side.properties_in[t].conc_mass_phase_comp['Liq', j]))
 
         @self.Constraint(self.flowsheet().config.time)
         def eq_over_pressure_ratio(b, t):
             return (b.feed_side.properties_out[t].pressure ==
                     b.over_pressure_ratio[t]
```

Comment:
Technically, we can make `over_pressure_ratio` an inlet/outlet variable too.

Reply:
Yeah it could be calculated but generally when people talk about over pressure they are just talking about where the driving force is the lowest. So I think making it be just the outlet makes sense.